packaging/packager_win.bat

```diff
@@ -0,0 +1,42 @@
+@Echo off
+
+SET DISTRO="windows"
+SET DISTRO_VERSION="10"
+SET DISTRO_ROOT="%HOMEDRIVE%"
+
+REM AutoPy is not available for Python3.9 at present
+SET DISABLE_AUTOPY=1
+REM Tesseract doesn't have original installers for Windows
+SET DISABLE_OCR=1
+REM Drag/drop with the current default DC backend is not supported on Windows
+SET DISABLE_DRAG=1
+REM Requires X server and thus only available on Linux
+SET DISABLE_XDOTOOL=1
+REM Requires VNC server and thus only available on Linux
+SET DISABLE_VNCDOTOOl=1
+
+REM Main deps
+cd %DISTRO_ROOT%\
+REM Assuming a local python executable to avoid overloading the download web page
+START /WAIT python-3.9.0-amd64.exe /quiet InstallAllUsers=1 PrependPath=1 Include_test=0
+REM set a temporary path variable valid inside this session
+set PYTHONPATH="C:\Program Files\Python39"
+set PATH=%PYTHONPATH%;%PYTHONPATH%\Scripts;%PATH%
+REM a permanent path variable will be set by the executable once this batch exits
+
+REM GuiBot deps
+pip install --user --upgrade pip
+pip install -r %DISTRO_ROOT%\guibot\packaging\pip_requirements.txt
+
+REM GuiBot setup
+echo Copying GuiBot files
+cd %DISTRO_ROOT%\guibot\packaging
+python setup.py install
+cd ..
+xcopy misc %PYTHONPATH%\Lib\site-packages\guibot\misc /E /S /I /Q /V
+xcopy tests %PYTHONPATH%\Lib\site-packages\guibot\tests /E /S /I /Q /V
+cd ..

```

Comment:
Is this `cd` call needed? Since you change directory again right after

Reply:
Indeed, this was preserved from the originally migrated script, I will also make it more compatible with the other packager scripts as
```diff
diff --git a/packaging/packager_win.bat b/packaging/packager_win.bat
index 2425a7a..9dfd907 100644
--- a/packaging/packager_win.bat
+++ b/packaging/packager_win.bat
@@ -32,10 +32,8 @@ REM GuiBot setup
 echo Copying GuiBot files
 cd %DISTRO_ROOT%\guibot\packaging
 python setup.py install
-cd ..
-xcopy misc %PYTHONPATH%\Lib\site-packages\guibot\misc /E /S /I /Q /V
-xcopy tests %PYTHONPATH%\Lib\site-packages\guibot\tests /E /S /I /Q /V
-cd ..
+xcopy %DISTRO_ROOT%\guibotmisc %PYTHONPATH%\Lib\site-packages\guibot\misc /E /S /I /Q /V
+xcopy %DISTRO_ROOT%\guibottests %PYTHONPATH%\Lib\site-packages\guibot\tests /E /S /I /Q /V
 
 REM Run all unit tests
 cd %PYTHONPATH%\Lib\site-packages\guibot\tests
```